packages/store/src/utils/persistTransformFilter.ts

```diff
@@ -0,0 +1,154 @@
+// this is copy-paste from https://github.com/edy/redux-persist-transform-filter/tree/master
+// I've changed to typescript and modified the tests to work with jest
```

Comment:
I decided to copy-paste the code as it is based on the transform https://github.com/rt2zz/redux-persist?tab=readme-ov-file#transforms, is not going to change often, is not typescript and I don't want to rely on yet another package.